### DISCUSSION TITLE
Cannot find any topics on Kafka

### DISCUSSION THREAD

**Discussion by iamjavadev1** [averemee-si/oracdc#48] [Q&A]
I have started the connector successfully but I can't find any topics on kafka after I update the data on the table.

<img width="1430" alt="Screen Shot 2566-08-08 at 13 30 19" src="https://github.com/averemee-si/oracdc/assets/14800892/532a78f4-d17e-459a-bfd0-15b351c31fc7">

for parameter to register connector
<img width="842" alt="Screen Shot 2566-08-08 at 13 33 08" src="https://github.com/averemee-si/oracdc/assets/14800892/04e12fcb-beb2-4b4f-9292-5e7473d18a05">

Screen Shot 2566-08-08 at 13 33 08
Regards,
Iamjavadev1

**Comment by averemee-si** [comment]
Hi Iamjavadev1,

To understand this I need full connect.log file from start of connector.

Regards,
Aleksei

**Reply by iamjavadev1** [reply]
Hi averemee-si,

This is my connect.log file. 
[connect.log](https://github.com/averemee-si/oracdc/files/12288710/connect.log)

Regards,
Iamjavadev1

**Comment by averemee-si** [comment]
Thanks,

Looks like no changes to capture in archived logs.
Please execute from sqlplus connected as user with DBA privileges
```
INSERT INTO SCOTT.EMP VALUES (8001,'TEST-EMP01','CLERK',7782,to_date('23-1-1982','dd-mm-yyyy'),1500,NULL,10);
commit;
alter system switch logfile;
```
then check for data in Kafka topic **EMP**

Regards,
Aleksei

**Comment by iamjavadev1** [comment]
It's worked.But I'm wondering if the changes will be sent to Kafka after I execute "alter system switch logfile". Why is that? And how to fix it so that I don't have to execute this command every time?

Regards,
Iamjavadev1

**Comment by averemee-si** [comment]
Hi Iamjavadev1,

This is Oracle RDBMS architecture question, not mine. In real life you does not need to execute
```
alter system switch logfile;
```
Oracle Database ARCnn processes will do this automatically **when** online redo is full and need archiving.

Regards,
Aleksei
